src/core/linux-aio.cc

```diff
@@ -170,8 +171,12 @@ int io_pgetevents(aio_context_t io_context, long min_nr, long nr, io_event* even
 void setup_aio_context(size_t nr, linux_abi::aio_context_t* io_context) {
     auto r = io_setup(nr, io_context);
     if (r < 0) {
-        char buf[1024];
-        char *msg = strerror_r(errno, buf, sizeof(buf));
+	char buf[1024];
```

Comment:
please replace <tab> with 4 spaces. see https://github.com/scylladb/seastar/blob/master/coding-style.md#whitespace

Reply:
done it